web/ui/react-app/src/pages/graph/Graph.tsx

```diff
@@ -98,8 +126,21 @@ class Graph extends PureComponent<GraphProps, GraphState> {
     const { chartData } = this.state;
     this.plot(
       this.selectedSeriesIndexes.length === 1 && this.selectedSeriesIndexes.includes(selectedIndex)
-        ? chartData.map(toHoverColor(selectedIndex, this.props.stacked))
-        : chartData.filter((_, i) => selected.includes(i)) // draw only selected
+        ? [...chartData.series.map(toHoverColor(selectedIndex, this.props.stacked)), ...chartData.exemplars]
+        : [
+            ...chartData.series.filter((_, i) => selected.includes(i)),
+            ...chartData.exemplars.filter(exemplar => {
+              series: for (const i in selected) {
+                for (const name in chartData.series[selected[i]].labels) {
+                  if (exemplar.labels[`Series: ${name}`] !== chartData.series[selected[i]].labels[name]) {
```

Comment:
It's a bit too surprising IMO that this depends on the exact magic string formatting (that one would assume would only be consumed by humans) from another place. I wonder if the series labels wouldn't better be stored in a separate set of labels (`seriesLabels`?) in the exemplar? But just thinking out loud here...

Reply:
Yup, that actually works great! It was my false belief that flot wouldn't allow any fields in the datapoint outside of its defined structure, but this isn't Go...